<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="name_edit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="number_edit"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="push_btn">
      <property name="text">
       <string>회원 등록</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>      이름</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>전화번호</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>91</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="previousbtn">
      <property name="text">
       <string>뒤로</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="create_btn">
      <property name="text">
       <string>엑셀 생성</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sendbtn">
      <property name="text">
       <string>일괄 전송</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_btn">
      <property name="text">
       <string>회원 삭제</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>341</width>
     <height>301</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>선택</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>전화번호</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
